index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://canvas-component-lib.netlify.app/Component.css">
+    <title>Document|Home</title>
+</head>
+<body>
+    <div class="desktop-container">
+      <img src="https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/nyfwsketches-1567608885.jpg?crop=1.00xw:1.00xh;0,0&resize=980:*" alt="Fashion-backdrop" class="responsive-img quiz-background-img" />
```

Comment:
The code looks clean. 
For the UI, I'd suggest you decrease the opacity of the bg image for better visibility of the button.

Reply:
Done